packages/next/src/client/components/router-reducer/ppr-navigations.ts

```diff
@@ -387,7 +387,10 @@ function createCacheNodeOnNavigation(
       // `prefetchRsc` field.
       rsc,
       prefetchRsc: null,
-      head: isLeafSegment ? possiblyPartialPrefetchHead : [null, null],
+      head: (isLeafSegment ? possiblyPartialPrefetchHead : null) ?? [
+        null,
+        null,
+      ],
```

Comment:
I'm a bit confused: shouldn't we either standardize on `null` representing `[null, null]`? Or consistently always do `[null, null]`?

```suggestion
      head: isLeafSegment ? possiblyPartialPrefetchHead : [null, null],
```

Reply:
`possiblyPartialPrefetchHead` is still nullable. I tried all in `[null, null]` first but feels it's bit redudant as prefetched data doesn't have to always hold a `[null, null]` as default value. When you want to check if it's nullable and fallback to current head, `[null, null]` would be a bit confusing to represent null and to check the values.

This will be a temeporary state, we'll have the [null, null] for short time then metadata will go to stay with page component. Then `head` and `prefetchedHead` will be back to single ReactNode